e2e/tests/e2e_happy_path/HappyPath.spec.ts

```diff
@@ -295,17 +286,21 @@ async function checkErrorMessageInApplicationController() {
     await previewWidget.waitAndSwitchToWidgetFrame();
     await previewWidget.waitAndClick(SpringAppLocators.springMenuButtonLocator);
     await previewWidget.waitAndClick(SpringAppLocators.springErrorButtonLocator);
-    await previewWidget.waitVisibility(SpringAppLocators.springErrorMessageLocator);
-    await driverHelper.getDriver().switchTo().defaultContent();
-}
 
-async function closeMainWindowAndSwitchToWorkspace(mainWindowHandle: string) {
-    await driverHelper.switchToSecondWindow(mainWindowHandle);
-    const secondWindowHandle: string = await driverHelper.getDriver().getWindowHandle();
-    await driverHelper.getDriver().switchTo().window(mainWindowHandle);
-    await driverHelper.getDriver().close();
-    await driverHelper.getDriver().switchTo().window(secondWindowHandle);
-    await driverHelper.getDriver().manage().window().setSize(TestConstants.TS_SELENIUM_RESOLUTION_WIDTH, TestConstants.TS_SELENIUM_RESOLUTION_HEIGHT);
+    try {
+        await previewWidget.waitVisibility(SpringAppLocators.springErrorMessageLocator);
+    } catch (err) {
+
```

Comment:
It's not clear why we can ignore an error here. Could you, please, add comment with explanation, and log it?